oak/common/app_config.cc

```diff
@@ -216,6 +258,11 @@ bool ValidApplicationConfig(const ApplicationConfiguration& config) {
     } else if (node.has_storage_node()) {
       in_ports[fqpn(node.node_name(), kStorageNodeRequestPortName)] = 0;
       out_ports[fqpn(node.node_name(), kStorageNodeResponsePortName)] = 0;
+      in_ports[fqpn(node.node_name(), kEscrowInPortName)] = 0;
+      out_ports[fqpn(node.node_name(), kEscrowOutPortName)] = 0;
```

Comment:
Does this imply that a storage node *must* have an associate escrow node?  The existence of a distinct `AddEscrowToConfig` implies not, but IIRC this will add a required-channel check at the end (I guess Travis will check my memory there, though).